### PR TITLE
chore: update readme with port forward example

### LINKED ISSUE #516: Update README and `solo node start` to provide commands to setup kubectl port forward for network nodes
requested by Simi for uses to be able to leverage to setup a connection against the network.

### PR DESCRIPTION
**Pull request by JeffreyDallas**
## Description

This pull request changes the following:

* update readme with port forward example

### Related Issues

* Closes #516

### CODE REVIEW

**Comment by JeffreyDallas** [comment]
Maybe we could proved a simple  `local_solo_launch.sh`

```
#!/usr/bin/env bash

./test/e2e/setup-e2e.sh

solo node keys --gossip-keys --tls-keys -i node1,node2,node3
solo network deploy -i node1,node2,node3
solo node setup -i node1,node2,node3
solo node start -i node1,node2,node3
solo mirror-node deploy
solo relay deploy -i node1

kubectl port-forward svc/relay-node1-hedera-json-rpc-relay -n solo-e2e 7546:7546 &
kubectl port-forward svc/haproxy-node1-svc -n solo-e2e 50211:50211 &
kubectl port-forward svc/fullstack-deployment-hedera-explorer -n solo-e2e 8080:80 &

```

File: README.md.template [hunk 1]
```diff
@@ -279,12 +279,26 @@ Once the nodes are up, you may now expose various services (using `k9s` (shift-f
 
 * Node services: `network-<node ID>-svc`
 * HAProxy: `haproxy-<node ID>-svc`
+  ```bash
+  # enable portforwarding for haproxy
+  kubectl port-forward svc/haproxy-node1-svc -n "${SOLO_NAMESPACE}" 50211:50211 &
```
Comment (jeromy-cannon) [inline README.md.template:284]:
```suggestion
  kubectl port-forward svc/haproxy-node1-svc -n "${SOLO_NAMESPACE}" 50211:50211 &
  kubectl port-forward svc/haproxy-node2-svc -n "${SOLO_NAMESPACE}" 51211:50211 &
  kubectl port-forward svc/haproxy-node3-svc -n "${SOLO_NAMESPACE}" 52211:50211 &
```

File: README.md.template [hunk 2]
```diff
@@ -279,12 +279,26 @@ Once the nodes are up, you may now expose various services (using `k9s` (shift-f
...
+  ```
 * Envoy Proxy: `envoy-proxy-<node ID>-svc`
+  ```bash
+  # enable portforwarding for haproxy
```
Comment (jeromy-cannon) [inline README.md.template:288]:
```suggestion
  # enable portforwarding for envoy proxy
```

File: README.md.template [hunk 3]
```diff
@@ -279,12 +279,26 @@ Once the nodes are up, you may now expose various services (using `k9s` (shift-f
...
+  kubectl port-forward svc/envoy-proxy-node1-svc -n "${SOLO_NAMESPACE}" 8081:8080 &
```
Comment (jeromy-cannon) [inline README.md.template:289]:
```suggestion
  kubectl port-forward svc/envoy-proxy-node1-svc -n "${SOLO_NAMESPACE}" 8081:8080 &
  kubectl port-forward svc/envoy-proxy-node2-svc -n "${SOLO_NAMESPACE}" 8181:8080 &
  kubectl port-forward svc/envoy-proxy-node3-svc -n "${SOLO_NAMESPACE}" 8281:8080 &
```

**Review by tomzhenghedera — APPROVED** [review]

**Review by jeromy-cannon — CHANGES_REQUESTED** [review]
please make the updates I requested

**Review by jeromy-cannon — APPROVED** [review]

**Pull request merged**